nomad_jobs/acs-interface.hcl

```diff
@@ -61,7 +63,7 @@ job "acs-interface" {
       }
       driver = "docker"
       config {
-        image = "vmck/acs-interface:interface"
+        image = "vmck/acs-interface"
```

Comment:
Let's tag versions so we know exactly what is running. Can you please tag a version already?